ReleaseProcessScript/tools/Core/main_helper_functions.ps1

```diff
@@ -69,27 +100,28 @@ function Get-Develop-Current-Version ($StartReleasebranch)
   {
     $MostRecentVersion = $DevelopVersion.Substring(1)
   }
-    
-  $PossibleVersions = Get-Possible-Next-Versions-Develop $MostRecentVersion $WithoutPrerelease
-  Write-Host "Please choose Release Version:"
-  $CurrentVersion = Read-Version-Choice $PossibleVersions
 
-  return $CurrentVersion
+  return $MostRecentVersion
 }
 
-function Get-Hotfix-Current-Version ($LastVersion, $StartReleasePhase)
+function Get-Hotfix-Most-Recent-Version ()
 {
-  if ($StartReleasePhase)
+  #Get current support branch from name and version of current branch
+  $CurrentBranchName = Get-Current-Branchname
+  $CurrentBranchFullVersion = Parse-Version-From-BranchName $CurrentBranchName
+  $CurrentBranchMajorMinorVersion = Get-Major-Minor-From-Version $CurrentBranchFullVersion
+  $CurrentSupportBranchName = "support/v$($CurrentBranchMajorMinorVersion)"
```

Comment:
I think this might be overkill. we already know the version for the hotfix branch. we don't need to get the tags from the support-branch as well.

Reply:
We don't get the tags from the support branch. But I agree, that might be a bit confusing. We need two pieces of information (the other two variables are auxiliary):

1. **Version of support branch**: This acts as fallback version if there is not at least one version tag on the hotfix branch (`else` branch of the `if` in this function).
2. **Name of support branch**: The support branch's name is required as `$To` (i. e. second) parameter in  
 `Get-Last-Version-Of-Branch-From-Tag-Exists "HEAD" $CurrentSupportBranchName`  
`Get-Last-Version-Of-Branch-From-Tag "HEAD" $CurrentSupportBranchName`. If there is a version tag in the "commit range" from HEAD (at this stage pointing to the hotfix branch) until the tip of the support branch, then return the highest version. This `$CurrentSupportBranchname` as second parameter prevents accidentally retrieving a version tag from "below" the fork point of the hotfix branch to release from.

Reply:
Okay, you need the support branch as the lower boundary of the search.